### PR TITLE
Feature/tank level

### PR DESCRIPTION
**Pull request by Hunter27**
Made api for heat map front end

### CODE REVIEW

File: WaterLog_Backend/Controllers/TankReadingsController.cs [hunk 1]
```diff
@@ -14,37 +14,50 @@ namespace WaterLog_Backend.Controllers
...
+        public TankReadingsController(DatabaseContext context, IConfiguration config)
         {
             _db = context;
             _config = config;
         }
 
         // GET api/levels
         [HttpGet]
-        public async Task<ActionResult<IEnumerable<TankLevelsEntry>>> Get()
+        public async Task<ActionResult<IEnumerable<TankReadingsEntry>>> Get()
```
Comment (FredJEhlers) [inline WaterLog_Backend/Controllers/TankReadingsController.cs]: Please use more descriptive end points

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Migrations/20190226074431_TankTables2.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+    public partial class TankTables2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "TimeStamp",
+                table: "TankReadings",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
```
Comment (FredJEhlers) [inline WaterLog_Backend/Migrations/20190226074431_TankTables2.cs:14]: This looks weird, why not use current timestamp or something like that and also why not specify the datetimekind

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Migrations/20190226074431_TankTables2.cs (hunk 2 above)
Comment (Hunter27) [reply WaterLog_Backend/Migrations/20190226074431_TankTables2.cs:14]: the current time stamp is used. We need all the readings with different timestamps for data collection and graph population

File: WaterLog_Backend/Controllers/TankReadingsController.cs (hunk 1 above)
Comment (Hunter27) [reply WaterLog_Backend/Controllers/TankReadingsController.cs]: This is just a simple get method? What more do i need to elaborate on?

Comment (FredJEhlers) [reply WaterLog_Backend/Controllers/TankReadingsController.cs]: What are you getting? 
`getTankReading` or `getTankblabla`
just `get` is not good enough

Comment (Hunter27) [reply WaterLog_Backend/Controllers/TankReadingsController.cs]: alright cool, made it more specific

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**